ci-operator/step-registry/assisted/baremetal/operator/ztp/add-day2-workers-optionally/assisted-baremetal-operator-ztp-add-day2-workers-optionally-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: assisted-baremetal-operator-ztp-add-day2-workers-optionally
+  from: assisted-service-scripts
+  commands: assisted-baremetal-operator-ztp-add-day2-workers-optionally-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 500Mi
+  timeout: 1h30m0s
+  grace_period: 5m0s
+  env:
+  - name: DISCONNECTED
+    default: "false"
+    documentation: By default we test in a connected environment.
+  - name: ASSISTED_ZTP_CONFIG
+    default: ""
+    documentation: Configuration for the add_day2_remote_nodes.sh script. See https://github.com/openshift/assisted-service/blob/master/deploy/operator/README.md for more details on these settings.
+  documentation: |-
+    Given a spoke cluster, add workers to it.
```

Comment:
Please document NUMBER_OF_DAY2_HOSTS here and explain that it's optional and doesn't add any workers when that variable is set to 0

Reply:
See lines above that document an env var...